src/features/admin/components/InvalidAdmin.tsx

```diff
@@ -0,0 +1,12 @@
+import { Markdown } from "~/components/ui/Markdown";
```

Comment:
I remember there's a hook called `isAdmin`, maybe is what you need?
https://github.com/privacy-scaling-explorations/maci-rpgf/blob/main/src/hooks/useIsAdmin.ts

could add this to protect the admin-only pages.